pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -116,70 +119,39 @@ func (b *Build) BuildPod(config BuildPodConfig, secrets []corev1.Secret, builder
 					Resources:       b.Spec.Resources,
 				},
 			},
+			SecurityContext: &corev1.PodSecurityContext{
```

Comment:
Why do we need this change?

Reply:
This allows us to configure the owner of the volume mounts and prevents us from needing to `chown` the volumes in the prepare step.